.github/workflows/continous-integration.yml

```diff
@@ -223,11 +223,15 @@ jobs:
         path: .venv
         key: ${{ runner.os }}-poetry-${{ env.POETRY_VERSION }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}-venv
 
+    - name: Clear Poetry cache
+      if: steps.cache-poetry.outputs.cache-hit == 'true' && needs.changes.outputs.backend == 'true' && contains(github.event.pull_request.labels.*.name, 'unit-tests:clear-poetry-cache')
```

Comment:
I'd advocate for `tools:clear-poetry-cache-unit-tests` that way we can have more tools like this. But I don't know enough about the current labels we use. I think are in a better position to choose

Reply:
I like it 👍🏻